src/js/debug.js

```diff
@@ -774,6 +774,7 @@ const DEBUG = {
         "Altitude",
         "RSSI",
         "Gyro",
+        "Attitude",
```

Comment:
```suggestion
```

For backwards compatibility: add attitude after line 836 (within semver block):
```
DEBUG.enableFields.splice(DEBUG.enableFields.indexOf("Gyro"), 0, "Attitude");
```

Reply:
I've suspected, that this improvement can not be too easy :)